pydantic_ai_slim/pydantic_ai/models/gemini.py

```diff
@@ -158,6 +159,13 @@ async def request(
             response = _gemini_response_ta.validate_json(data)
         return self._process_response(response)
 
+    async def count_tokens(
+        self,
+        messages: list[ModelMessage],
+    ) -> BaseCountTokensResponse:
+        """Token counting is not supported by the CohereModel."""
+        raise NotImplementedError('Token counting is not supported by CohereModel')
```

Comment:
Wrong model name, but should be fixed by making the super method non-abstract and dropping the definition here